infrastructure/clear-all-data.sh

```diff
@@ -129,6 +129,7 @@ docker run --rm --network=$NETWORK --entrypoint=/bin/sh minio/mc:RELEASE.2025-05
 POSTGRES_DB="events"
 EVENTS_MIGRATOR_ROLE="events_migrator"
 EVENTS_APP_ROLE="events_app"
+ANALYTICS_POSTGRES_USER=${ANALYTICS_POSTGRES_USER:-"events_analytics"}
```

Comment:
What's the role of this user in deployed environments or even locally? I just see it being dropped if it exists